backend/api/Areas/Admin/Models/Parcel/ParcelModel.cs

```diff
@@ -10,7 +10,7 @@ public class ParcelModel : Model.BaseModel
 
         public string PID { get; set; }
 
-        public string PIN { get; set; }
+        public int? PIN { get; set; }
```

Comment:
Fixing possible future bugs here so that the type is correct.  I'm not aware of the PIN formatting, so we may need to make it a string in the database.